crates/cursor-core/src/auth/mod.rs

```diff
@@ -68,11 +68,13 @@ pub async fn sign_in() {
     }
 
     let uuid = Uuid::new_v4().to_string();
+    console::log_str(&format!("zhangzhuang uuid: {}", uuid));
```

Comment:
these logs should be removed

Reply:
sorry about that, I forget to remove this logs, it's very difficult when I want to debug the Rust-part code, I will remove this and update the PR.